crates/next-core/src/next_shared/webpack_rules/babel.rs

```diff
@@ -54,41 +53,44 @@ pub async fn detect_likely_babel_loader(
 pub async fn get_babel_loader_rules(
     project_root: FileSystemPath,
 ) -> Result<Vec<(RcStr, LoaderRuleItem)>> {
-    let mut has_babel_config = false;
+    let mut babel_config_path = None;
     for &filename in BABEL_CONFIG_FILES {
-        let filetype = *project_root.join(filename)?.get_type().await?;
+        let path = project_root.join(filename)?;
+        let filetype = *path.get_type().await?;
         if matches!(filetype, FileSystemEntryType::File) {
-            has_babel_config = true;
+            babel_config_path = Some(path);
             break;
         }
     }
-    if !has_babel_config {
+    let Some(babel_config_path) = babel_config_path else {
         return Ok(Vec::new());
-    }
+    };
 
-    if !*is_babel_loader_available(project_root.clone()).await? {
-        BabelIssue {
-            path: project_root.clone(),
-            title: StyledString::Text(rcstr!(
-                "Unable to resolve babel-loader, but a babel config is present"
-            ))
-            .resolved_cell(),
-            description: StyledString::Text(rcstr!(
-                "Make sure babel-loader is installed via your package manager."
-            ))
-            .resolved_cell(),
-            severity: IssueSeverity::Fatal,
-        }
-        .resolved_cell()
-        .emit();
-    }
+    // - See `packages/next/src/build/babel/loader/types.d.ts` for all the configuration options.
+    // - See `packages/next/src/build/get-babel-loader-config.ts` for how we use this in webpack.
+    let mut loader_options = serde_json::Map::new();
```

Comment:
use the json! macro?

Reply:
`options` is typed specifically as `serde_json::Map` and not `serde_json::Value`, but this seems reasonable (with a forced downcast).